docs/source/quick_start.rst

```diff
@@ -34,11 +34,25 @@ the *keyring*. Here's an example of a decrypted keyring file ::
 
     {
       "lochness": {
+        "REDCAP": {
```

Comment:
This spelling should be changed to `REDCap`. Actually, what code looks for the `REDCAP` word, can you point me?

Reply:
https://github.com/PREDICT-DPACC/lochness/blob/52103a14b8d56019bf25d64247264946435db16a/lochness/redcap/__init__.py#L311-L344